app/src/main/java/com/example/tasky/feature_agenda/presentation/event_detail_screen/EventDetailViewModel.kt

```diff
@@ -303,8 +317,18 @@ class EventDetailViewModel @Inject constructor(
 
             val existingEventId = state.value.eventId
             existingEventId?.let {
-                val eventToBeUpdated = eventToBeCreated.copy(eventId = it)
-                useCases.event.updateEvent(eventToBeUpdated)
+                val eventToBeUpdated = eventToBeCreated.copy(
+                    eventId = it,
+                    host = state.value.eventCreatorId,
+                    attendees = attendeeList.map { attendee ->
+                        attendee.copy(eventId = it)
+                    }.also { attendeeList ->
+                        attendeeList.distinctBy { attendee ->
+                            attendee.userId
+                        }
```

Comment:
How about adding a validation instead that attendees can only be added if that attendee doesn't exist for the event yet? Otherwise, it might be confusing for the user if you could add an attendee multiple times